src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const KeyAndValues = sourceString.split(';');
+  const end = {};
```

Comment:
The variable name 'end' is not descriptive of its purpose. Consider renaming it to something more indicative of its role, such as 'styleObject' or 'cssProperties'.